### ISSUE TITLE
Special case for working on a single file script

### ISSUE THREAD

**Issue by ethercrow** [schell/steeloverseer#24]
When I'm working on a script named "foo.py", I'd like to use `sos foo.py` as a shorthand for `sos foo.py -c foo.py`. The former invocation does nothing, so this change should not break anything.

**Comment by schell** [comment]
I'm a bit overwhelmed with work at the moment, but I would accept a PR for this feature ;)

**Comment by mitchellwrosen** [comment]
I assume you'd (always?) rather run `./file` when `file` is touched, in case `.` is not in your PATH.

**Comment by schell** [comment]
I'm not so sure about this request as it saves only a little typing. @ethercrow is there any other reason you prefer not to write `foo.py -c ./foo.py`?

**Comment by ethercrow** [comment]
No, only repetition. I could easily make a shell shorthand `sos1` for this, so no worries.

**Comment by mitchellwrosen** [comment]
I'm slightly in favor, as you're right it doesn't clash with any syntax. Currently, running `sos` without any `-c` arguments (and no yaml file) is basically a mistake :)
